src/main/java/io/tus/java/client/TusUploader.java

```diff
@@ -268,6 +274,18 @@ public URL getUploadURL() {
         return uploadURL;
     }
 
+    public void setProxy(Proxy proxy) {
```

Comment:
Can you please add a documentation for the method here?

Reply:
Done!